.buildkite/scripts/publish-distribution.sh

```diff
@@ -21,7 +21,9 @@ docker buildx create --use
 # do not push if DRY_RUN is true
 if [[ ${DRY_RUN:-true} == "true" ]]; then
     docker buildx imagetools create --dry-run -t "${DOCKER_IMG_TARGET}" "${DOCKER_IMG_SOURCE}"
+    docker buildx imagetools create --dry-run -t "${DOCKER_IMG_TARGET}-ubi" "${DOCKER_IMG_SOURCE}-ubi"
 else
     retry 3 docker buildx imagetools create -t "${DOCKER_IMG_TARGET}" "${DOCKER_IMG_SOURCE}"
+    retry 3 docker buildx imagetools create -t "${DOCKER_IMG_TARGET}-ubi" "${DOCKER_IMG_SOURCE}-ubi"
```

Comment:
:+1: 
Just to confirm, this would re-tag the docker images:
- docker.elastic.co/package-registry/distribution:lite-ubi
- docker.elastic.co/package-registry/distribution:production-ubi 

as the following ones respectively:
- docker.elastic.co/package-registry/distribution:lite-X.Y.Z-ubi
- docker.elastic.co/package-registry/distribution:X.Y.Z-ubi